chart/templates/replicated-supportbundle.yaml

```diff
@@ -19,9 +19,36 @@ stringData:
     spec:
       collectors:
         - logs:
-            collectorName: replicated-logs
+            collectorName: replicated-sdk-logs
             selector:
               {{- range $k, $v := (include "replicated.labels" . | fromYaml) }}
               - {{ $k }}={{ $v }}
-              {{- end }} 
-            name: replicated/logs
+              {{- end }}
+            name: replicated-sdk/logs
+        - http:
+            collectorName: replicated-sdk-app-info
+            get:
+              url: http://replicated-sdk.{{ include "replicated.namespace" . }}:3000/api/v1/app/info
```

Comment:
Given that we support both names, we should use `{{ include "replicated.serviceName" . }}` here to keep it dynamic